bodyshopbidsdotcom/git-scripts
# Add support for CircleCI for AWS git-scripts

## What

To run AWS related git-scripts like `git_push_eb` on circleci, we need to disable MFA token and use `default` aws profile.

If there is `CIRCLECI` env variable set, then `default` aws profile will be used which doesn't need MFA token to be entered. I have created IAM user `circleci` which doesn't have MFA enabled.

## Testing
With `CIRCLECI`:
1. Run commands like `CIRCLECI=true git_push_eb qa6` 
2. It should not ask MFA token and should display `Using aws profile name default`

Without `CIRCLECI`:
1. Run commands like `git_push_eb qa6` 
2. It should ask MFA token if token is expired and should display `Using aws profile name user_session`
